src/Faker/Provider/ru_RU/Address.php

```diff
@@ -69,12 +69,47 @@ class Address extends \Faker\Provider\Address
         'Наро-Фоминск', 'Ногинск', 'Одинцово', 'Озёры', 'Орехово-Зуево',
         'Павловский Посад', 'Подольск', 'Пушкино', 'Раменское', 'Сергиев Посад',
         'Серебряные Пруды', 'Серпухов', 'Солнечногорск', 'Ступино', 'Талдом',
-        'Чехов', 'Шатура', 'Шаховская', 'Щёлково',
+        'Чехов', 'Шатура', 'Шаховская', 'Щёлково', 'Благовещенск', 'Владивосток', 
+        'Иркутск', 'Хабаровск', 'Чита', 'Белогорск', 'Красноярск', 'Находка', 
```

Comment:
This package complies with the [PSR-2 standard](https://github.com/fzaninotto/Faker/blob/master/CONTRIBUTING.md), it can 't have a [trailing whitespace at the end of the line](https://www.php-fig.org/psr/psr-2/#23-lines)

Please run `make sniff test` before you commit changes. You can use `phpcbf` to fix and comply to PSR-2, or manually remove the trailling whitespaces. The tests will pass after this fix.